app/javascript/components/GlobalHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+const GlobalHeader = () => {
+  return (
+    <nav className='navbar navbar-expand bg-dark'>
+      <div className='container-fluid'>
+        <Link to="/" className='navbar-brand'>teaDB</Link>
+        <Link to="/login">Login</Link>
+        <Link to="/signup">Signup</Link>
+        <Link to="signout">Signout</Link>
```

Comment:
I think you can remove this for now, since we will only show this when the user is signed in. That can happen in a future PR.